cypress/component/DataSearch/dataset_search_table.spec.js

```diff
@@ -38,6 +41,31 @@ describe('Dataset Search Table tests', () => {
       cy.get('#header-checkbox').click();
       cy.contains('1 dataset selected from 1 study');
     });
+  });
+
+  describe('Data library filter by participant count tests', () => {
+    beforeEach(() => {
+      cy.initApplicationConfig();
+      cy.stub(TerraDataRepo, 'listSnapshotsByDatasetIds').returns({});
+    });
+
+    it('When a participant count filter is applied the query is updated', () => {
+      var filtered = false;
+      function handler(request) {
+        if (JSON.stringify(request.body).includes('{"range":{"participantCount":{"gte":null,"lte":"50"}}}')) {
+          filtered = true;
+        }
+        request.reply({statusCode: 200, body:[]});
+      }
+
+      cy.intercept(
+        {method: 'POST', url: '**/api/dataset/search/index'}, handler).as('searchIndex');
+      mount(<DatasetSearchTable {...props} />);
+      cy.get('#participantCountMax-range-input').clear().type('50');
+      cy.wait(1000).then(() => {
```

Comment:
when I wait on 'searchIndex' it doesn't work I think because there is another call to this api without the filters applied first